src/app/results/results.component.css

```diff
@@ -418,27 +418,38 @@ div.autocorrect {
 }
 
 @media screen and (max-width: 560px) and (min-width: 495px) {
-  #settings{
+  #settings {
     left: 27%;
   }
-  #tools{
+  #tools {
     left: 27%;
   }
 }
 
 @media screen and (max-width: 495px) {
-  #settings{
-    left: 10%;
+  #settings {
+    left: 0;
   }
-  #tools{
-    display:none;
+  #tools {
+    left: 0;
   }
 }
-@media screen and (max-width: 420px) {
-  #settings{
-    left: 5%;
+ @media screen and (max-width: 387px) {
+  #search-options li {
+    padding: 0 2vw 12px!important;
+  }
+  }
+
+  @media screen and (max-width: 327px) {
+    #search-options li {
+      padding: 0 1vw 12px!important;
+    }
+    }
+@media screen and (max-width: 283px) {
+    #search-options li {
+      padding: 0 0vw 12px!important;
```

Comment:
Needs to be made more appropriate and codacy fails for it also.

Reply:
Codacy is complaining about using important which is already used at many places. There is no problem with that.

Reply:
@praveenojha33 I think Codacy states not to use it. And use same units for any single property CSS.

Reply:
`important` is already used at many places in project.

Reply:
Could you fix Codacy?

Reply:
I will have to remove `important` but it will take more time and effort for no reason.